packages/app/src/app/overmind/effects/vscode/SandboxFsSync/index.ts

```diff
@@ -404,30 +404,36 @@ class SandboxFsSync {
     autoInstallTypes: boolean
   ) {
     try {
-      if (
-        autoInstallTypes &&
-        this.typesInfo[dep.name] &&
-        !dep.name.startsWith('@types/')
-      ) {
-        const name = `@types/${dep.name}`;
-        this.fetchDependencyTypingFiles(name, this.typesInfo[dep.name].latest)
-          .then(files => {
-            this.setAndSendPackageTypes(dep.name, files);
-          })
-          .catch(e => {
-            if (process.env.NODE_ENV === 'development') {
-              console.warn('Trouble fetching types for ' + name);
-            }
-          });
-      }
-
       try {
         const files = await this.fetchDependencyTypingFiles(
           dep.name,
           dep.version
         );
+        const hasTypes = Boolean(
+          Object.keys(files).some(
+            key => key.startsWith(dep.name) && key.endsWith('.d.ts')
```

Comment:
Laast nitpick, apparently it needs be this (checked https://ata.codesandbox.io/api/v8/react%40latest)

```suggestion
            key => key.startsWith(`/${dep.name}`) && key.endsWith('.d.ts')
```